packages/librewolf-app/librewolf-app.pacscript

```diff
@@ -14,23 +14,19 @@ sha256sums_arm64=(
   "SKIP"
   "SKIP"
 )
-if [[ ${CARCH} == amd64 ]]; then
-  gnuarch="x86_64"
-else
-  gnuarch="aarch64"
-fi
 source=(
-  "https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.${gnuarch}.AppImage"
   "icon.png::https://gitlab.com/uploads/-/system/group/avatar/5407097/icon.png"
   "${gives}.desktop::https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/${pkgname}/${gives}.desktop"
 )
+source_amd64=("https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/${pkgver}/LibreWolf.x86_64.AppImage")
```

Comment:
this one needs to deprecate the old source array and just double it up for each CARCH. Remember CARCH will get appended *after* regular var

Reply:
Can't the order of the sums be switched instead?

Reply:
logistically speaking, yes that would work. But for maintainability, the primary source should always come first.